calc_flow_v000.ipynb

```diff
@@ -0,0 +1,514 @@
+    "    design_current = current * 1.25"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DC\n",
+      "1\n",
+      "27.91\n"
+     ]
+    }
+   ],
+   "source": [
+    "design_current = truncate(design_current,2)\n",
+    "print(voltage_type)\n",
+    "print(phase_count)\n",
+    "print(design_current)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter the conductor material (CU or AL): CU\n",
+      "CU\n"
+     ]
+    }
+   ],
+   "source": [
+    "cond_material = str(input('Enter the conductor material (CU or AL): '))\n",
+    "print(cond_material)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100, 110, 125, 150, 175, 200, 225, 250, 300, 350, 400, 450, 500, 600, 700, 800, 1000, 1200, 1600, 2000, 2500, 3000, 4000, 5000, 6000]\n"
+     ]
+    }
+   ],
+   "source": [
+    "ocpd_check = 0\n",
+    "ocpd_check = list(nec_tables.ocpd_sizes)\n",
+    "print(ocpd_check)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Are you using an OCPD that is not listed in NEC 240.4? (Y/N): N\n"
+     ]
+    }
+   ],
+   "source": [
+    "ocpd_custom = str(input('Are you using an OCPD that is not listed in NEC 240.4? (Y/N): '))\n",
+    "if ocpd_custom == 'Y':\n",
+    "    custom = int(input('Enter the non-standard OCPD rating (A): '))\n",
+    "    ocpd_check.insert(0,custom)\n",
+    "    print(ocpd_check)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100, 110, 125, 150, 175, 200, 225, 250, 300, 350, 400, 450, 500, 600, 700, 800, 1000, 1200, 1600, 2000, 2500, 3000, 4000, 5000, 6000]\n"
+     ]
+    }
+   ],
+   "source": [
+    "ocpd_check.sort()\n",
+    "print(ocpd_check)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "37\n"
+     ]
+    }
+   ],
+   "source": [
+    "list_range = len(ocpd_check)\n",
+    "print(list_range)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in ocpd_check:\n",
```

Comment:
With this syntax i is not the integer location of the list.  It is the value of the list.  If you wanted to get an integer python has the handy enumerate() function, but in this case you can just use the returned value and skip the indexing.

Change i to something like `ocpd_size`.

Reply:
Corrected. It worked - thanks!